sqlglot/generator.py

```diff
@@ -2083,9 +2083,22 @@ def query_modifiers(self, expression: exp.Expression, *sqls: str) -> str:
             self.sql(expression, "order"),
             *offset_limit_modifiers,
             *self.after_limit_modifiers(expression),
+            *self.sql(expression, "option"),
```

Comment:
So if you changed the `"option"` to `"options"` then you could change this line to:

```python
self.expressions(expression, key="options")
```

Reply:
Done

Reply:
Is there any way to wrap the csv result in parentheses when generating the sql, if using expressions?

Reply:
You can either use `self.wrap(self.expressions(...))` or manually wrap it using an f-string, like `f"({...})"`. The former may be better because it formats the wrapped csv when `pretty=True`. You can play with it a bit and see if it works as expected, otherwise use the f-string wrapping as a fallback?

Reply:
This would mean I can't use the `prefix` option for the `OPTION` - I think I would need to resort to using `wrap` + f string, and checking whether the `OPTION` actually exists - therefore I think something like this would work:
```
        option_sql = ""
        if expression.args.get("options", []):
            option_sql = f" OPTION{self.wrap(self.expressions(expression, key='options'))}"
```

WDYT?

Reply:
Makes sense - this probably needs to be done before the `return` statement is reached. Here's how I'd write this:

```python
options = self.expressions(expression, key="options")
if options:
    options = f" OPTION{self.wrap(options)}"
```